pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -119,6 +120,12 @@ func (dpi *GenericDevicePlugin) Start(stop <-chan struct{}) (err error) {
 		devnode, err := os.Open(dpi.devicePath)
 		if err == nil {
 			devnode.Close()
+		} else {
+			cmd := exec.Command("modprobe", dpi.deviceName)
```

Comment:
I am not sure that it is correct to assume the presence of `modprobe` tool in the container

```console
$ podman run --rm -ti --entrypoint bash quay.io/kubevirt/virt-handler:v1.1.1
bash-5.1# modprobe
bash: modprobe: command not found
bash-5.1#
```

Reply:
Besides, as I read through the referenced issue from Flatcar, I am not yet convinced that it should be fixed in KubeVirt.

Reply:
Thank you for the review, @vasiliy-ul - I will close this PR for now, until there is a better approach or a fix in the Flatcar.